docs/_tutorials/rehearsal/rehearsal.rst

```diff
@@ -0,0 +1,70 @@
+Rehearsal
+---------
+
+A popular and efficient method in continual learning is to do **rehearsal**. Aka
+the action of reviewing a limited amount of previous images.
+
+Continuum provides a helper class to manage this memory and functions to
+samples ("to herd") examples into the memory.
+
+How to use the memory:
+
+.. code-block:: python
+
+    from torch.utils.data import DataLoader
+
+    from continuum import ClassIncremental
+    from continuum.datasets import CIFAR100
+    from continuum import rehearsal
```

Comment:
I think that this folder  architecture looks like we propose a full strategy, while we only propose a memory tool.
I believe RehearsalMemory should be in the same folder as TaskSet

Reply:
We also propose functions to do herding. And this module may grow, so I don't really agree.

Reply:
Herding functions can be used for other strategies / reason than rehearsal, and it could be useful on taskset object directly